src/node.ts

```diff
@@ -101,7 +102,11 @@ async function bundle(
       ...(isBadCommonJs(input) ? [(virtual as any)({"-": shimCommonJs(input, require)})] : []),
       importResolve(input, cacheRoot, packageRoot),
       nodeResolve({browser: true, rootDir: packageRoot}),
-      (commonjs as any)({esmExternals: true}),
+      json(),
+      (commonjs as any)({
+        esmExternals: true,
+        requireReturnsDefault: "preferred"
```

Comment:
This was the only value of the property that allowed us to import elliptic without breaking imports of react. 😞 I don’t really understand this, but it’s described here:

https://github.com/rollup/plugins/tree/master/packages/commonjs/#requirereturnsdefault

My hand-wavy understanding is that we ideally want the default `requireReturnsDefault: false`. However, this assumes that the required module is compatible with cjs-module-lexer, and thus Rollup is able to convert the CommonJS exports into named exports. But often cjs-module-lexer doesn’t work, and so we can’t know whether a `require("foo")` will be converted into a default export or named exports. The *preferred* setting lets us use the default export is used if it’s available, and otherwise we use named exports. (The *auto* setting didn’t work in the case of the [brorand](https://www.npmjs.com/package/brorand) package, whose CommonJS is converted into both a named export and a default export.)

I’m sure there remain plenty of cases where you still can’t import CommonJS, but this seems like a more compatible setting than the default.